### PR TITLE
Replace IRC with bridged channels

### PR DESCRIPTION
**Pull request by NotMyFault**
IRC channels are now bridged to matrix, which is bridged to gitter.
Benefits are, that it's easier to access, without the need to set up a bouncer to stay connected or read past messages, unless you are already using matrix.

### CODE REVIEW

**Review by timja — COMMENTED** [review]
can we use matrix instead? gitter usernames are awful on IRC

image

image

**Comment by NotMyFault** [comment]
> can we use matrix instead? gitter usernames are awful on IRC

For sure, the imposter names are an odd choice by the bridge bot.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
-  - name: "❓ IRC"
-    url: https://web.libera.chat/
-    about: Please ask and answer questions here, (#jenkins-release on libera.chat).
```
Comment (daniel-beck) [inline .github/ISSUE_TEMPLATE/config.yml:5]: While the Matrix link is useful for those who use it or would use webchat, this basically makes the link useless to anyone who's not using it, like the old "click here" antipattern.

File: .github/ISSUE_TEMPLATE/1-lts-release-checklist.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ This role should rotate between LTS releases
 
 - [ ] Assure that contents of master branch have been merged to the stable branch in the [packaging repository](https://github.com/jenkinsci/packaging), e.g. `stable-2.361`
 
-- [ ] Announce the start of the LTS release process in the #jenkins-release and #jenkins-infra IRC channels
+- [ ] Announce the start of the LTS release process in the [#jenkins-infra IRC channels](https://matrix.to/#/#jenkins-infra:libera.chat).
```
Comment (daniel-beck) [inline .github/ISSUE_TEMPLATE/1-lts-release-checklist.md:73]:
```suggestion
- [ ] Announce the start of the LTS release process in the [#jenkins-infra IRC channel](https://matrix.to/#/#jenkins-infra:libera.chat).
```

File: .github/ISSUE_TEMPLATE/1-lts-release-checklist.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ This role should rotate between LTS releases
 
 - [ ] Assure that contents of master branch have been merged to the stable branch in the [packaging repository](https://github.com/jenkinsci/packaging), e.g. `stable-2.361`
 
-- [ ] Announce the start of the LTS release process in the #jenkins-release and #jenkins-infra IRC channels
+- [ ] Announce the start of the LTS release process in the [#jenkins-infra IRC channel](https://matrix.to/#/#jenkins-infra:libera.chat).
```
Comment (lemeurherve) [inline .github/ISSUE_TEMPLATE/1-lts-release-checklist.md:73]: No need to announce anymore on #jenkins-release too?

Comment (NotMyFault) [reply .github/ISSUE_TEMPLATE/1-lts-release-checklist.md:73]: The announcement is for the infra team (only afaik), I find the infra channel is the most appropriate one then, no?

Comment (lemeurherve) [reply .github/ISSUE_TEMPLATE/1-lts-release-checklist.md:73]: Sure, that's a great recent addition we're appreciating, just wondering about the #jenkins-release annoucement as it was part of the original checklist, and this PR would remove it.

Comment (NotMyFault) [reply .github/ISSUE_TEMPLATE/1-lts-release-checklist.md]: Added it again 👍🏻

File: .github/ISSUE_TEMPLATE/1-lts-release-checklist.md [hunk 4]
```diff
@@ -70,8 +70,8 @@ This role should rotate between LTS releases
 
 - [ ] Assure that contents of master branch have been merged to the stable branch in the [packaging repository](https://github.com/jenkinsci/packaging), e.g. `stable-2.361`
 
-- [ ] Announce the start of the LTS release process in the #jenkins-release and #jenkins-infra IRC channels
-
+- [ ] Announce the start of the LTS release process in the [#jenkins-release](https://matrix.to/#/#jenkins-release:libera.chat) and [#jenkins-infra](https://matrix.to/#/#jenkins-infra:libera.chat) IRC channels
+-
```
Comment (lemeurherve) [inline .github/ISSUE_TEMPLATE/1-lts-release-checklist.md:74]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -1,5 +1,5 @@
...
+  - name: "💬 Chat"
+    url: https://matrix.to/#/#jenkins-release:libera.chat
+    about: Please ask and answer questions here.
```
Comment (lemeurherve) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: Please ask and answer questions here.
  - name: "❓ IRC"
    url: https://web.libera.chat/
    about: Click here to access IRC from a browser (#jenkins-release on libera.chat).
```
Keep the previous IRC link but at the end.

**Review by lemeurherve — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 6]
```diff
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
+  - name: "💬 Chat"
+    url: https://matrix.to/#/#jenkins-release:libera.chat
+    about: Please ask and answer questions here.
   - name: "❓ IRC"
     url: https://web.libera.chat/
-    about: Please ask and answer questions here, (#jenkins-release on libera.chat).
+    about: Click here to access IRC from a browser (#jenkins-release on libera.chat).
```
Comment (timja) [inline .github/ISSUE_TEMPLATE/config.yml:8]: click here doesn't make sense as this text isn't hyperlinked

File: .github/ISSUE_TEMPLATE/config.yml [hunk 7]
```diff
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
+  - name: "💬 Chat"
+    url: https://matrix.to/#/#jenkins-release:libera.chat
+    about: Please ask and answer questions here.
   - name: "❓ IRC"
```
Comment (timja) [inline .github/ISSUE_TEMPLATE/config.yml:6]: I don't think we need IRC and chat links? can't we just have the matrix one.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 8]
```diff
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
+  - name: "💬 Chat"
+    url: https://matrix.to/#/#jenkins-release:libera.chat
+    about: Please ask and answer questions here.
```
Comment (timja) [inline .github/ISSUE_TEMPLATE/config.yml:5]: maybe one of 
```suggestion
    about: Please ask and answer questions on Matrix (or one of the bridges e.g. libera IRC or gitter) (#jenkins-release).
```

```suggestion
    about: Please ask and answer questions on Matrix (#jenkins-release).
```

File: .github/ISSUE_TEMPLATE/config.yml (hunk 7 above)
Comment (NotMyFault) [reply .github/ISSUE_TEMPLATE/config.yml:6]: Daniel wants to keep the IRC one, see the comment above. But in my opinion, matrix-only is fine.

Comment (timja) [reply .github/ISSUE_TEMPLATE/config.yml:6]: can you link to the comment? I'm not seeing it

File: .github/ISSUE_TEMPLATE/config.yml (hunk 8 above)
Comment (NotMyFault) [reply .github/ISSUE_TEMPLATE/config.yml:5]: I'd be in favor of the second one.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 7 above)
Comment (NotMyFault) [reply .github/ISSUE_TEMPLATE/config.yml:6]: https://github.com/jenkins-infra/release/pull/289#discussion_r974677491

Comment (timja) [reply .github/ISSUE_TEMPLATE/config.yml:6]: I think that was referring to the text `Please ask and answer questions here` i.e. text not being useful. re-including #jenkins-release should fix that.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 6 above)
Comment (NotMyFault) [reply .github/ISSUE_TEMPLATE/config.yml:8]: How about `If you don't use Matrix, connect via IRC on libera.chat` ?

Comment (timja) [reply .github/ISSUE_TEMPLATE/config.yml:8]: could do although I think we could just remove it, most people who aren't already in are likely to prefer matrix.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 8 above)
Comment (lemeurherve) [reply .github/ISSUE_TEMPLATE/config.yml]: The first mentions IRC though, I'd go with it as it's more descriptive.

**Review by timja — APPROVED** [review]

**Pull request merged**